tests/testthat/test-po.R

```diff
@@ -11,13 +11,21 @@ test_that("translation domain correctly loaded", {
 })
 
 test_that("modified translations are correctly reloaded", {
+  # Need to also specify `LC_ALL` because `LANGUAGE` is ignored when
+  # `LANG` is set (here via `LC_ALL`) to `C` or `C.UTF-8`
```

Comment:
Should this go upstream into withr too?

Reply:
Here is the upstream change: https://github.com/r-lib/withr/commit/687f3a39c603622aea3bc54c3decbb379972b0f3

It's only giving a warning as the caller needs to set the locale.